src/app/technical-record/body/body-psv/body-psv.component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { ComponentFixture, getTestBed, TestBed } from '@angular/core/testing';
+import { SharedModule } from '@app/shared/shared.module';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { TESTING_UTILS } from '../../../utils/testing.utils';
+import { BodyPsvComponent } from './body-psv.component';
+import { TechRecord } from '@app/models/tech-record.model';
+
+describe('PsvBodyComponent', () => {
+  let component: BodyPsvComponent;
+  let fixture: ComponentFixture<BodyPsvComponent>;
+  let injector: TestBed;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [SharedModule],
+      declarations: [BodyPsvComponent],
+      schemas: [CUSTOM_ELEMENTS_SCHEMA]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(BodyPsvComponent);
+    injector = getTestBed();
+    component = fixture.componentInstance;
+    component.activeRecord = TESTING_UTILS.mockTechRecord({
+      vehicleType: 'psv',
+      bodyMake: 'test',
+      bodyModel: 'test',
+      modelLiteral: 'test',
+      chassisMake: 'volvo',
+      chassisModel: 't23'
+    });
+    
+    fixture.detectChanges();
+  });
+
+  it('should create view only with populated data', () => {
+    fixture.detectChanges();
```

Comment:
Something minor:
Not sure we need this again, we are calling it in the beforeEach() block already and we are not manipulating the component again by adding/modifying data so we don't need to call it again I believe.